server/datastore/mysql/migrations/tables/20220915153947_HostDisplayName.go

```diff
@@ -0,0 +1,40 @@
+package tables
+
+import (
+	"database/sql"
+	"github.com/pkg/errors"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20220915153947, Down_20220915153947)
+}
+
+func Up_20220915153947(tx *sql.Tx) error {
+	for _, change := range []struct{ name, sql string }{
+		{"delete index", `ALTER TABLE hosts DROP INDEX hosts_search`},
+		{"create index", `CREATE FULLTEXT INDEX hosts_search ON hosts(hostname, uuid, computer_name)`},
```

Comment:
Does the order of the columns in a fulltext index impact performance in a similar fashion as other indexes? If so, should we consider modifying the order here, perhaps to put `computer_name` first?

Reply:
There should be no difference AFAIK.